eventdata/challenges/document_id_benchmark.json

```diff
@@ -26,7 +26,7 @@
         "operation-type": "create-index-template",
         "settings": {
             "index.translog.retention.size": "10mb",
-            "index.number_of_shards": {{ shard_count | default(1) }}
+            "index.number_of_shards": {{ number_of_shards | default(1) }}
```

Comment:
The change from the default is not documented. However, I'm ok if we do this in a follow-up PR.